bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubHandler.java

```diff
@@ -113,7 +113,7 @@ public void initialize() {
             logger.debug("hub '{}' port={}", getThing().getUID(), config.portNumber);
         }
 
-        if (config.portNumber <= 0 || config.portNumber > 0xFFFF) {
+        if (config.portNumber != PORT_TCP && config.portNumber != PORT_WSS) {
```

Comment:
This seems too strict, it means you can no longer configure the port to anything but default values for TCP/secure web socket.

Reply:
I made it so strict, because the hub actually has those two ports (only) hard coded in the hub. I suppose you could imagine an edge case where a user system had some kind of port forwarding from other port numbers, but it is hardly a common use case.

Reply:
And to be more specific, the communication protocol depends on which of the two ports are used. If the client connects on the TCP port, it runs the TCP protocol; if it connects on the WSS port, it runs the WSS protocol; and if tries to connect on any other port, the connection is refused.

Reply:
In that case the configuration seems backwards. Would it make sense to configure which protocol to use, then use appropriate port accordingly? In that case you could even have the protocol configurable, since you would know the protocol not from the port. And port forwarding configurations would then be supported.

Reply:
Good point.

Reply:
Done